tcms/testruns/static/testruns/js/search.js

```diff
@@ -90,7 +122,7 @@ $(document).ready(function() {
                         result += '<p class="help-block">' + data.stop_date + '</p>';
                     }
                     return result;
-                }
+        }
```

Comment:
unrelated formatting change, revert it.  Also actually breaks the existing formatting.